src/core/grpc_service.proto

```diff
@@ -188,6 +194,62 @@ message HealthResponse
   bool health = 2;
 }
 
+//@@
+//@@.. cpp:var:: message ControlRequest
+//@@
+//@@   Request message for Control gRPC endpoint.
+//@@
+message ControlRequest
+{
+  //@@  .. cpp:enum:: Type
+  //@@
+  //@@     Types of control operation
+  //@@
+  enum Type {
+    //@@    .. cpp:enumerator:: Type::UNLOAD = 0
+    //@@
+    //@@       To unload the specified model.
+    //@@
+    UNLOAD = 0;
+
+    //@@    .. cpp:enumerator:: Type::LOAD = 1 << 0
```

Comment:
Using "1 << 0" implies that this is a bit-mask where multiple control options can be specified. I don't think that is the intent.